website/docs/components/time/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,24 @@
+## Conformance rating
+
+<!-- Update conformance rating badge with correct status and remove the others -->
+<Doc::Badge @type="success">Conformant</Doc::Badge>
+
+When used as recommended, there should not be any WCAG conformance issues with this component.
+
+<Doc::Badge @type="warning">Conditionally conformant</Doc::Badge>
```

Comment:
Issue: need to remove this conditionally conformant/not conformant badges and sample text (lines 8-14)

Reply:
Well... thinking about it now -I guess the component is conditionally conformant. as long as you put it in the right text component and dont make the text contrast too low its conformant. I don't know how @MelSumner prefers to document this kind of nuance though, so I defer to her.

Reply:
Updating to conformant for now as I don't see a reason why it wouldn't be unless abused in some strange way.